web/annotation_layer_builder.js

```diff
@@ -97,7 +97,7 @@ class AnnotationLayerBuilder {
    * @returns {Promise<void>} A promise that is resolved when rendering of the
    *   annotations is complete.
    */
-  async render(viewport, options, intent = "display") {
+  async render(viewport, options, intent = "display", linkAnnotations) {
```

Comment:
The new parameter is not documented, also the name should be clearer in that these are not "normal" LinkAnnotations; e.g.
```suggestion
  async render(viewport, options, intent = "display", inferredLinkAnnotations) {
```